core/baseline-core-v1.0-psd01.md

```diff
@@ -3001,60 +3003,227 @@ This section of the document focuses on the requirements and considerations rela
+
+Preconditions:
+
+* A PBI test system is up and running.
+* There is a given set of test BPI Subjects that are part of a test workgroup.
+* The test workgroup is associated with only one test workstep.
+* There exists a system of record identified as the single source for the input data.
+* A test workstep has been created that requires internal authoritative input data from a single source.
+* The BPI Subjects involved in the workstep have agreed that the identified system of record is the authoritative source for the input data.
+* The test workstep accepts only input data from the agreed upon data source for a given data schema, and the input data is transformed in an agreed upon manner to an expected output.
+* The agreed-upon input data source is configured with a test data set.
+
+Test Steps:
+
+1. Retrieve the input data for the identified BPI workstep from the single source system of record.
+2. Verify that the retrieved input data matches the expected format and is consistent with the requirements of the test workstep.
+3. Verify that the single source system of record is the authoritative source for the input data in the test workstep.
+
+Test Passing Criteria:
+
+* The retrieved input data matches the expected format and is consistent with the requirements of the test workstep.
+* The identified single source system of record is confirmed to be the authoritative source for the input data.
 
 ## 8.2 External Authoritative Data for BPIs
 
+###### extauthbpidata
+
 External authoritative data means that the input data to a BPI workstep is held in some authoritative 3rd party database, such as government records. Unlike internal authoritative data where there is only one source, external authoritative data input to a BPI workstep may have multiple sources.
 
-#### **[R315]** 
+#### **[R314]** 
 BPI Subjects participating in a workstep MUST agree upon the source and type of the external authoritative data used as input to a BPI workstep.
 
-[[R315]](#r15) Testability: An agreement on authoritative data sources will be embedded in a commercial document electronically represented on a BPI as required by [[R6]](#r6) to be bdining. And since [[R6]](#r6) is testable, [[R315]](#r15) is testable.
+[[R314]](#r314) Testability: 
+
+Preconditions:
+
+* There is a given set of test BPI Subjects that are part of a test workgroup.
+* The test workgroup is associated with only one test workstep.
+* There exists an external system identified as the single source for the input data.
+* A test workstep has been created that requires external authoritative input data from a single source.
+* The test workstep accepts only input data from the agreed upon data source for a given data schema, and the input data is transformed in an agreed upon manner to an expected output.
+* The agreed-upon input data source is configured with a test data set.
+* An agreement quorum has been defined and configures in the test BPI.
+
+Test Steps:
```

Comment:
There should be at least two tests here - one that demonstrates that nothing happens if you don't get an agreement (i.e. quorum), and another that it works if (and only when) you do.

Reply:
added two preconditions on configured BPI behavior if a quorum is reached and if it is not reached.